cogment_verse/replay_buffers/torch_replay_buffer.py

```diff
@@ -78,11 +178,11 @@ def sample(self, num):
             indices = self._rng.choice(self.size(), size=num, replace=False)
 
         return TorchReplayBufferSample(
-            observation=self.observation[indices],
-            next_observation=self.next_observation[indices],
-            action=self.action[indices],
-            reward=self.reward[indices],
-            done=self.done[indices],
```

Comment:
Thank you for reminding me this. I need to do the same for other buffer. It is because of the safety precaution. I think they store at the same memory pointer. It would be bad if we do any manipulation with this observation. You can test it using this function. 
```python
import torch
def samestorage(x,y):
    if x.storage().data_ptr()==y.storage().data_ptr():
        print("same storage")
    else:
        print("different storage")
```